.github/workflows/snarkyjs-api-reference.yml

```diff
@@ -0,0 +1,56 @@
+name: SnarkyJS-API-Reference
+on:
+  workflow_dispatch:
+    snarkyjs_branch:
+      type: choice
+      description: SnarkyJS branch to build from
```

Comment:
Cool, I didn't know 'choice' was an option in Github Actions.

What is the benefit you envision here? Wouldn't we want to always consistently follow whichever branch matches the published version of SnarkyJS (i.e. `releases` curretly)?

Reply:
At the time, I figured that it might be useful if we made a documentation change in SnarkyJS but the more I think about it, I think it should just be one branch (releases) as you suggest. I'll fix this in the PR.